server/server.js

```diff
@@ -154,6 +154,10 @@ Server.prototype._processMessage = function(socket, message) {
     return;
   }
 
+  if (!this._notLimited(socket, message, messageType, resource)) {
+    return;
+  }
+
```

Comment:
Make call to *_limited* function (rename to emphasize positive logic).  Yes, you were following the pattern of the other calls in this function, but *_limited* is easier to read and understand.